docs/lib/graphqlapi/fragments/native_common/authz/common.md

```diff
@@ -1,9 +1,29 @@
 For client authorization AppSync supports API Keys, Amazon IAM credentials, Amazon Cognito User Pools, and 3rd party OIDC providers. This is inferred from the `amplifyconfiguration.json` file when you call `Amplify.configure()`.
 
-## API Key
+#### API Key
```

Comment:
API key (no reason to title case this as it isn't a proper noun)

Reply:
```suggestion
#### API key
```